### PR TITLE
Framework renamed to g-cloud

### PR DESCRIPTION
**Pull request by robyoung**
See: https://github.com/alphagov/digitalmarketplace-api/pull/336

### CODE REVIEW

File: app/main/suppliers.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def suppliers_list_by_prefix():
     page = process_page(request.args.get('page', default=u"1"))
 
     try:
-        api_result = data_api_client.find_suppliers(api_prefix, page, 'gcloud')
+        api_result = data_api_client.find_suppliers(api_prefix, page, 'g-cloud')
```
Comment (TheDoubleK) [inline app/main/suppliers.py]: Should we be properly paramaterize these supplier listing pages with `<framework>` in the route while we're here?  Or we don't know if we're going to want these pages for DOS at all yet? (From discussion so far it sounds like we will want something that allows people to see the suppliers on DOS).

Comment (robyoung) [inline app/main/suppliers.py]: I think this should go in as is. It's fixing a specific bug and we're not absolutely sure we know how the DOS pages are going to work yet.

Comment (TheDoubleK) [inline app/main/suppliers.py]: OK fine, agreed.

**Comment by TheDoubleK** [comment]
:+1:

**Pull request merged**